Hello_Microverse.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
+</html>
```

Comment:
- [ ] It will be better if you will give a class name for the `h1` tag and style it in `Styles.css` file.